packages/react/src/components/iconCard/IconCard.tsx

```diff
@@ -57,11 +57,20 @@ export interface IconCardProps {
      * @default false
      */
     disabled?: boolean;
+    /**
+     * Whether to place the badges above the title.
+     */
+    placeBadgesAbove?: boolean;
+    /**
+     * Classes to apply to the card button element.
+     */
+    cardClasses?: string[];

```

Comment:
I would tend to use something closer to React naming, something like `cardClassName` and make it a string. In the admin you can use className if you need something more complex
```suggestion
    /**
     * Additional CSS class to apply to the card button element.
     */
    cardClassName?: string;
```

Reply:
Yeah totally could. It would fit the React className typing. When initially deciding I did a word search in the project and landed on other component that define it as an array and went with that. I think it was the BlankSlate which is where I also got the original name of `cardClasses`.
I'll change it because I agree your suggestion is better.

Reply:
I just realized you already made that change in your suggestion and I let it slip. Oups!

Reply:
👍 
While we worked on the sandbox/documenting props we saw a lot of those classes `string[]` and we think it's weird to use compared to `className`